waterloop-site/src/components/NavPopup/NavPopup.tsx

```diff
@@ -0,0 +1,83 @@
+import React, {useState} from "react";
+import styled from 'styled-components';
+import {Close} from '@styled-icons/evil';
+
+const Card = styled.div`
+    position: absolute;
+    right: 20px;
+    padding: 20px;
+    background: #F4F4F4;
+    width: 300px;
+    height: 165px;    
+    border-radius: 6px;
+    z-index: 100000;
+    align-items: center;
+    font-size: 15px;
+    opacity: 0;
+
+    animation: fadeIn 2.5s;
+    animation-delay: 0.3s;
+    animation-fill-mode: forwards;
+    @keyframes fadeIn {
+      0% {opacity:0;}
+      100% {opacity:1;}
+    }
+`;
+
+const Title = styled.div`
+    font-size: 24px;
+    margin-bottom: 15px;
+`;
+
+export const CloseIcon = styled(Close)`
+  color: black;
+  width: 30px;
+  position: absolute;
+  right: 20px;
+  top: 20px;
+  &:hover {
+    color: grey;
+    cursor: pointer;
+  }
+`;
+
+const Button = styled.button`
+    margin-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+    height: 37px;
+    font-weight: bold;
+    font-size: 16px;
+    color: white;
+    display: block;
+    background: #84B274;
+    border: none;
+    border-radius: 100px;
+    padding: 8px 16px;
+    &:hover {
+        background: #3E6232;
+        transition: background-color 0.2s ease-in;
+    }
+`;
+
+
+const NavPopup: React.FC = () => {
+    const [isOpen, setIsOpen] = useState(true);
+
+    function closePopup() {
+        setIsOpen(false);
+    }
+
+    if (!isOpen) { return null; }
+    return (
+        <Card>
+            <Title>Sustainable Tech Event</Title>
+            <CloseIcon onClick={closePopup}/>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+            <Button onClick={(): Window | null => window.open("/sustainable-tech", "_self")}>Check it out!</Button>
```

Comment:
Can you actually implement this functionality using React Router instead? I would suggest using `Link`.

`import { Link } from 'react-router-dom';`